jsapp/js/actions.es6

```diff
@@ -375,8 +375,12 @@ actions.resources.updateAsset.listen(function(uid, values, params={}) {
         actions.resources.updateAsset.completed(asset);
         resolve(asset);
       })
-      .fail(function(...args){
-        reject(args)
+      .fail((args) => {
+        if (args.status === 500) {
+          reject(args)
+        } else {
+          alertify.error(t(args.responseText));
```

Comment:
Should we still` reject(args)` after `alertify.error()` or is`alertify.error()` a Promise that is rejected?

Reply:
Thanks for catching this.
Alertify is not a promise. We use `alertify.error` here to generate the nice looking browser pop ups to showcase the error to the user.
I called reject to complete the async operation after notifying the user.